alex/applications/thub.py

```diff
@@ -213,10 +220,11 @@ def run(self):
 
         except KeyboardInterrupt:
             print 'KeyboardInterrupt exception in: %s' % multiprocessing.current_process().name
-            self.close_event.set()
+            #self.close_event.set()
             return
         except:
             self.cfg['Logging']['system_logger'].exception('Uncaught exception in THUB process.')
+            #self.close_event.set()
```

Comment:
Which 4 rows? This is a mistake introduced in https://github.com/UFAL-DSG/alex/commit/e97f1fc7938ad25f1681e27149533ea953a7a583 I will delete the lines.